src/main/resources/application.properties

```diff
@@ -1,3 +1,4 @@
+haveno.price-node.api.version=1.0.0
```

Comment:
To match the other properties, thinking this should be haveno.price.api.version or simply haveno.price.version, depending if we use "/api/version" or "/version".

Reply:
changed to haveno.price.version